.github/workflows/codeql.yml

```
@@ -0,0 +1,96 @@
+  schedule:
+    - cron: '29 19 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze Go
+    # Runner size impacts CodeQL analysis time. To learn more, please see:
+    #   - https://gh.io/recommended-hardware-resources-for-running-codeql
+    #   - https://gh.io/supported-runners-and-hardware-resources
+    #   - https://gh.io/using-larger-runners (GitHub.com only)
+    # Consider using larger runners or machines with greater resources for possible analysis time improvements.
+    runs-on: 'ubuntu-latest' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)